[2025-09-08T17:09:25.479+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:09:25.487+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-04-12T00:00:00+00:00 [queued]>
[2025-09-08T17:09:25.492+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-04-12T00:00:00+00:00 [queued]>
[2025-09-08T17:09:25.492+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:09:25.503+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): taskBranch> on 2025-04-12 00:00:00+00:00
[2025-09-08T17:09:25.507+0000] {standard_task_runner.py:72} INFO - Started process 20633 to run task
[2025-09-08T17:09:25.510+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskBranch', 'backfill__2025-04-12T00:00:00+00:00', '--job-id', '152', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmph1oxr1ph']
[2025-09-08T17:09:25.511+0000] {standard_task_runner.py:105} INFO - Job 152: Subtask taskBranch
[2025-09-08T17:09:25.542+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-04-12T00:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:09:25.585+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskBranch' AIRFLOW_CTX_EXECUTION_DATE='2025-04-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-04-12T00:00:00+00:00'
[2025-09-08T17:09:25.585+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:09:25.595+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-09-08T17:09:25.599+0000] {python.py:240} INFO - Done. Returned value was: taskC
[2025-09-08T17:09:25.599+0000] {branch.py:38} INFO - Branch into taskC
[2025-09-08T17:09:25.600+0000] {skipmixin.py:230} INFO - Following branch {'taskC'}
[2025-09-08T17:09:25.605+0000] {skipmixin.py:278} INFO - Skipping tasks [('taskE', -1)]
[2025-09-08T17:09:25.624+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:09:25.624+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskBranch, run_id=backfill__2025-04-12T00:00:00+00:00, execution_date=20250412T000000, start_date=20250908T170925, end_date=20250908T170925
[2025-09-08T17:09:25.644+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:09:25.672+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:09:25.676+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
